src/rgw/rgw_dedup.cc

```diff
@@ -1401,6 +1427,7 @@ namespace rgw::dedup {
         }
       }
 
+      p_stats->ingress_slabs++;
```

Comment:
is the ingress slabs change related to throttling?

Reply:
It helped debug the permission issue mentioned above (SLAB writes were failing because of using the wrong application permission).